app/views/full-page-examples/what-is-your-nationality/index.njk

```diff
@@ -62,8 +63,11 @@
             {
               id: "confirm-nationality",
               value: "british-nationality",
-              text: "British (including English, Scottish, Welsh or from Northern Ireland)",
-              checked: values["confirm-nationality"]|length and "british-nationality" in values["confirm-nationality"]
+              text: "British",
+              checked: values["confirm-nationality"]|length and "british-nationality" in values["confirm-nationality"],
```

Comment:
I think the second checkbox we can move the text in brackets into hint text too.

Reply:
👍